### PR TITLE
Fixing a11y issues.

### PR DESCRIPTION
**Pull request by jugshaurya**
This PR will fix up issue #54

**Solution**
- adding span with role "img"
- adding alt attribute to img tags
- adding rel="noopener noreferrer".

### CODE REVIEW

**Comment by jugshaurya** [comment]
@pranshuchittora, please review

**Comment by Uyadav207** [comment]
@jugshaurya, I have been working on the issue for 2 two days, kindly find some other. Thanks.

please have a look at #47

File: src/containers/Drawer/index.jsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ function ResponsiveDrawer(props) {
   const [selectedIndex, setSelectedIndex] = React.useState(-1);
 
   const handleDrawerToggle = (value) => {
-    if(value === false){
```
Comment (tewarig) [inline src/containers/Drawer/index.jsx:80]: it is done by prettier ? ( adding spaces don't help...)

File: src/containers/Drawer/index.jsx [hunk 2]
```diff
@@ -130,8 +133,7 @@ function ResponsiveDrawer(props) {
             aria-label="open drawer"
             edge="start"
             onClick={handleDrawerToggle}
-            className={classes.menuButton}
```
Comment (tewarig) [inline src/containers/Drawer/index.jsx:133]: same here as well

File: src/containers/Drawer/index.jsx (hunk 1 above)
Comment (jugshaurya) [reply src/containers/Drawer/index.jsx:80]: Logic wise it doesn't but definitely improves readability.

**Comment by pranshuchittora** [comment]
Already fixed in a PR before this. 
Duplicate https://github.com/react-native-elements/playground/pull/47

Feel free to reopen if there's no overlap

**Pull request closed without merging**